src/main/scala/org/ergoplatform/nodeView/history/extra/ExtraIndexSerializer.scala

```diff
@@ -0,0 +1,51 @@
+package org.ergoplatform.nodeView.history.extra
+
+import scorex.core.serialization.ScorexSerializer
+import scorex.util.serialization.{Reader, Writer}
+
+object ExtraIndexSerializer  extends ScorexSerializer[ExtraIndex]{
+
+    override def serialize(obj: ExtraIndex, w: Writer): Unit = {
+      obj match {
+        case m: IndexedErgoAddress =>
+          w.put(IndexedErgoAddress.modifierTypeId)
```

Comment:
Better not to use ModifierTypeId for extra index objects, in #1940 ModifierTypeId reworked into NetworkObjectTypeId, so this type is for objects sent over the wire ( except of ErgoFullBlock, but that will be reworked )

Reply:
done